contributors.md

```diff
@@ -0,0 +1,73 @@
+<html>

```

Comment:
The contributors details are not according to [all-contributors specs](https://github.com/kentcdodds/all-contributors)